tests/utils/test_audio_utils.py

```diff
@@ -20,19 +20,21 @@
 
 from transformers.audio_utils import (
     amplitude_to_db,
-    chroma_filter_bank,
+    amplitude_to_db_batch,
     hertz_to_mel,
     mel_filter_bank,
     mel_to_hertz,
     power_to_db,
+    power_to_db_batch,
     spectrogram,
+    spectrogram_batch,
     window_function,
 )
-from transformers.testing_utils import is_librosa_available, require_librosa
+from transformers.testing_utils import is_librosa_available
 
 
 if is_librosa_available():
-    from librosa.filters import chroma
+    pass
```

Comment:
Why are we checking if librosa is available if we don't import anything at the end of the day ?

Reply:
I think you removed a test by mistake !